_rules/SC4-1-2-role-attribute-has-valid-value.md

```diff
@@ -0,0 +1,123 @@
+---
+name: Role attribute has valid value
+test_type: atomic
+
+description: |
+   This rule checks that role attributes have valid values
+   
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any [non-empty](#non-empty) `role` attribute that is placed on an HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies).
+
+### Expectation
+
+Each test target has at least one non-abstract valid value as specified in the [WAI-ARIA 1.1, roles](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+Older browsers do not support fallback roles. If multiple values are used in the role attribute, the attribute is ignored.
```

Comment:
I think fallback roles need an explanation here, or a link to one.

Reply:
@annethyme I think I commented on another PR, where you had a similar suggestion, and I had no avail to find a link for `fallback` role. 

Please feel free to recommend.

Reply:
Suggestion:

> "Older browsers do not support more than one token in the value for a role attribute. If multiple values are used in the role attribute, the attribute is ignored in these browsers.
> 
> **Note:** Having a whitespace separated list of more than one token in the value of the role attribute is used for what is known as "fallback roles". If the first token is not accessibility supported (or valid), the next one will be used for determining the [semantic role](#semantic-role) of the element, and so forth."